ban-importer/src/core.js

```diff
@@ -190,15 +191,19 @@ export default class Core {
       );
 
       try {
-        if (exportBan.status === 'TO_BE_CREATED') await Core.createExportBan(exportBan);
-        else await Core.deleteExportBan(exportBan);
+        if(!erroringLists.includes(exportBan.ExportBanList.id)) {
+          if (exportBan.status === 'TO_BE_CREATED')
+            await Core.createExportBan(exportBan);
+          else await Core.deleteExportBan(exportBan);
+        }
       } catch (err) {
+        erroringLists.push(exportBan.ExportBanList.id);
         Logger.verbose(
           'Core',
           1,
           `Failed to ${
             exportBan.status === 'TO_BE_CREATED' ? 'create' : 'delete'
-          } export ban (ID: ${exportBan.id}): `,
+          } export ban (ID: ${exportBan.id}) - Ignoring exports for List ${exportBan.ExportBanList.id} for this run: `,
```

Comment:
Is there a way that this can throw an error in logging so we can take action on it with PM2?